migrator/database/tests/test_postgresql.py

```diff
@@ -4,19 +4,19 @@
 
 from psycopg2 import OperationalError 
 
-from database.PostgreSQLDatabase import PostgreSQLDatabase 
-from server.PostgreSQLServer import PostgreSQLServer
+from migrator.database.PostgreSQLDatabase import PostgreSQLDatabase 
+from migrator.server.PostgreSQLServer import PostgreSQLServer
 
 
 logging.basicConfig(
-    filename='TestDatabase_PostgreSQL.log',
+    filename='TestServer_PostgreSQL.log',
```

Comment:
Filename here should not change. Should be reverted to `TestDatabase_PostgresSQL.log`.